### ISSUE TITLE
Author Metadata: {Phil Sidney Ostheimer}

### ISSUE THREAD

**Issue by PhilSid** [acl-org/acl-anthology#3532]
### Author Pages

Hi,

could you please merge my author pages https://aclanthology.org/people/p/phil-ostheimer/ and https://aclanthology.org/people/p/phil-sidney-ostheimer/ into the latter one (https://aclanthology.org/people/p/phil-sidney-ostheimer/)?

### Type of Author Metadata Correction

- [ ] The author page wrongly conflates different people with the same name.
- [X] This author has multiple pages with different spellings of their name.
- [ ] This author has permanently changed their name.

### Supporting Information

Hi, my middle name (Sidney) is somehow missing from https://aclanthology.org/people/p/phil-ostheimer/ 
Therefore, please merge the page into https://aclanthology.org/people/p/phil-sidney-ostheimer/

Best

Phil Sidney Ostheimer

**Comment by PhilSid** [comment]
Hi, when will this be fixed?

**Comment by PhilSid** [comment]
@anthology-assist

**Comment by mbollmann** [comment]
@PhilSid I see that your middle name is also not present on the PDF of https://aclanthology.org/2024.lrec-main.1373/, so the correct fix would actually be to remove it from the metadata of that paper (2024.lrec-main.1373), so that your only author page will be https://aclanthology.org/people/p/phil-ostheimer/ @anthology-assist 

(It’s Anthology policy that the metadata should match what’s in the PDF, as it’s the only way to ensure consistency of citations)

**Comment by PhilSid** [comment]
Thanks for the reply @mbollmann !
So how can I change the metadata for that pdf/paper as it is already published? Also could I add my second name to the other paper https://aclanthology.org/2023.findings-acl.687/ instead?

**Comment by mbollmann** [comment]
All metadata is stored in this repo in XML format, for example in https://github.com/acl-org/acl-anthology/blob/master/data/xml/2024.lrec.xml for LREC 2024. You can find your paper and make the change there yourself and open a pull request, or you can wait until @anthology-assist gets around to it.

If you want to use your second name, you would need to provide updated versions of the PDFs and open new issues to request a paper revision for them.

**Comment by mjpost** [comment]
Hello: we are attempting to close out a large backlog of author page requests. As part of these efforts, we are collecting additional information ([ORCID](https://orcid.org/) and degree institution) which will help us assign papers to the correct author in the future. Please modify the updated description above with the requested information.

If you are requesting to split an author page (i.e., your page has some papers that are not yours), please also provide a list of your papers, in the form of Anthology IDs or URLS (e.g., 2023.wmt-1.13 or https://aclanthology.org/2023.wmt-1.13/). We are unable to match papers to Google or Semantic Scholar profiles.

**Comment by PhilSid** [comment]
Hi @mjpost this is still not resolved. There are still two author pages for me

- https://aclanthology.org/people/phil-ostheimer/
- https://aclanthology.org/people/phil-sidney-ostheimer/
I would still prefer if both would be merged into https://aclanthology.org/people/phil-sidney-ostheimer/
Best
Phil

**Comment by mjpost** [comment]
Yes, did you see the note above? We need your ORCID and institution.

**Comment by mjpost** [comment]
I can merge #5822 once I have that information. Sorry about the delays! We are severely backlogged on a number of issues like this, but are on the cusp of catching up.

**Comment by mjpost** [comment]
Ping @PhilSid

**Comment by PhilSid** [comment]
@mjpost I've just created my Orchid account: https://orcid.org/0009-0009-6186-3233
my institution is RPTU Kaiserslautern-Landau
